docs/source/delayed.rst

```diff
@@ -209,3 +209,14 @@ on the side-effect of another task. In these cases you can use
 ``sum_data`` will operate on DATA only after both the expected items have
 been appended to it. ``bind`` can also be used along with direct dependencies
 passed through the function arguments.
+
+
+Execution
+---------
+
+By default, Dask Delayed uses the threaded scheduler in order to avoid data
+transfer costs.  
```

Comment:
Do we know this is the reason why? Asking because I do not know...

Reply:
There you got me, I have to admit I just copy/pasted this from Dask Array documentation. I also know this is true, avoiding inter process data exchange can surely optimize a computation.

But in the end, I'm not sure if this is the real reason why Delayed uses the threaded scheduler by default.